charts/opensearch-operator/values.yaml

```diff
@@ -45,6 +48,13 @@ manager:
   # watch objects in the desired namespace. Defaults is to watch all namespaces.
   watchNamespace:
 
+# Install the Custom Resource Definitions with Helm
+installCRDs: true
+
+serviceAccount:
```

Comment:
why that client will choose to not create SA?

Reply:
Because in high-security clusters, you need to strictly manage everything, including service accounts.
Actually, it should also be possible to use `rbac.create` (or similar) so the users could handle it themselves.